### ISSUE TITLE
dssm loss计算为什么是reduce_sum

### ISSUE THREAD

**Issue by liamlmy** [InsaneLife/dssm#23]
在dssm.py中，计算loss的代码
with tf.name_scope('Loss'):
    # Train Loss
    cross_entropy = tf.nn.sigmoid_cross_entropy_with_logits(labels=doc_label_batch, logits=cos_sim)
    losses = tf.reduce_sum(cross_entropy)
    tf.summary.scalar('loss', losses)
    pass
是不是有问题？为什么是reduce_sum?而不是reduce_mean
